modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -200,15 +200,15 @@ $(document).ready(function() {
 
         let $target = $(event.currentTarget);
         let tabName = '';
-        let $tabPanel = $($target).attr('data-target');
```

Comment:
@ayumihamsaki you're misled by @bennothommo comment a little bit that should be left alone.
When you get attr it's not a jQuery object anymore and will throw an error here `$($target).attr('aria-selected', 'true');`

Reply:
what about this line:

let $tabPanel = $($target).attr('data-target');